modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -388,7 +388,10 @@ public static enum ENUM_PROPERTY_NAMING_TYPE {camelCase, PascalCase, snake_case,
     public static final String DISALLOW_ADDITIONAL_PROPERTIES_IF_NOT_PRESENT_DESC =
         "If false, the 'additionalProperties' implementation (set to true by default) is compliant with the OAS and JSON schema specifications. " +
         "If true (default), keep the old (incorrect) behaviour that 'additionalProperties' is set to false by default.";
-
+    public static final String UNSUPPORTED_V310_SPEC_MSG =
```

Comment:
should this be removed as well?

Reply:
Nope it is used for the warning logging. It could be moved into defaultcodegen only

Reply:
👌